nginx/config.go

```diff
@@ -186,6 +186,9 @@ http {
 		location / {
 			return 404;
 		}
+		location /nginx_status {
+			stub_status on;
+		}
```

Comment:
You still want this to be conditional though, yes? Also, should this `location` (but not the entire default vhost) be either locked down to a configurable whitelist, or possibly protected with a configurable username/password. I tend to imagine we don't want to expose these stats to the world.